.github/workflows/on_pull_request.yml

```diff
@@ -418,6 +418,11 @@ jobs:
         run: yarn test:website
         env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+          APPLITOOLS_BATCH_NAME: "${{ github.event.pull_request.head.repo.full_name }}: pull request from ${{ github.event.pull_request.head.ref }} - ${{ github.event.pull_request.title }} (${{ github.event.pull_request.head.sha }})"
+          GITHUB_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
```

Comment:
Lines 422 and 423 were ported from the github workflow for storybook VRT when it was still using applitools. 
Source: https://github.com/twilio-labs/paste/pull/1259/files#diff-4177c38233ccde70569c4fb02d752aaec30c3279d95a00c8af8f2382e7c3bc95